dev/tests/integration/framework/Magento/TestFramework/Annotation/DbIsolation.php

```diff
@@ -17,6 +20,85 @@ class DbIsolation
      */
     protected $_isIsolationActive = false;
 
+    /**
+     * @var string[]
+     */
+    private $dbStateTables = [
+        'catalog_product_entity' => 'assertIsEmpty',
+        'eav_attribute' => 'eavAttributeAssert',
+        'catalog_category_entity' => 'assertTwoRecords',
+        'eav_attribute_set' => 'attributeSetAssert',
+        'store' => 'assertTwoRecords'
+    ];
+
+    /**
+     * @param array $data
```

Comment:
Could you provide any description to the method?
It's unclear what's its responsibility

Reply:
It is not important at the moment

Reply:
Let me stabiliize it first